modules/custom/d_content/modules/d_content_init/src/ContentInitManagerBlock.php

```diff
@@ -60,7 +90,9 @@ protected function createBlockContent(array $block) {
       'info' => $block['info']['title'],
     ]);
     $this->processFields($block,$block_entity);
-    $this->placeBlockContent($block, $block_entity);
+    if ($placed_block = $this->placeBlockContent($block, $block_entity)) {
+      $this->placeBlockInWaMegaMenu($block, $placed_block);
```

Comment:
so, every block now will be placed in MegaMenu ?
this solution doesn't allow us placing some custom block in some other region? or will it be changed when footer blocks will be reworked?

Reply:
Block will be placed in mega menu only if YML contain **we_megamenu** declaration. See: `placeBlockInWaMegaMenu()` method.